packages/odata-v4/src/request/odata-bound-action-request-config.ts

```diff
@@ -3,10 +3,10 @@ import {
   EntityBase,
   RequestMethodType,
   EntityApi,
+  OperationParameters,
   WithKeys
 } from '@sap-cloud-sdk/odata-common/internal';
 import { DeSerializers } from '../de-serializers';
-import { ActionImportParameters } from './action-import-parameter';
```

Comment:
[req] The file name and class name differ. I think if we remove the "import" everywhere this should be fine. 😊

Reply:
This line is removed in this PR. I think having `OperationParameters` is fine for now?

Reply:
This was not about this line but the filename and class name (but there is no way to annotate the whole file). I think you already did it though.

Reply:
They should be consistent now :)